foreign.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+const foreign = require('foreign');
+const readFile = (filename, callback) => fs.readFile(filename, {encoding: 'utf8'}, callback);
+const relative = (filename) => path.join(process.argv[2], filename);
+
+const main = () => {
+  const complete = (error, files) => {
+    error ?
+      process.stderr.write(error.message + '\n') :
+      process.stdout.write(files.join(''));
+
+    process.exit(error ? 1 : 0);
```

Comment:
Rather than branch in two places, I'd prefer to see:

```javascript
if (err == null) {
  process.stdout.write(files.join(''));
  process.exit(0);
} else {
  process.stderr.write(err.message + '\n');
  process.exit(1);
}
```